src/engraving/rendering/dev/tlayout.cpp

```diff
@@ -4821,7 +4821,7 @@ static void layoutStemSlash(const StemSlash* item, const LayoutContext& ctx, Ste
             && (musicFont == "Bravura" || musicFont == "Finale Maestro" || musicFont == "Gonville")) {
             angle *= angleIncrease;
         }
-        endX = hook->layoutData()->bbox().translated(hook->pos()).right(); // always ends at the right bbox margin of the hook
+        endX = hook->layoutData()->bbox().translated(hook->pos()).right() - hook->offset().x(); // always ends at the right bbox margin of the hook
```

Comment:
Maybe it would be better to replace `hook->pos()` with `hook->layoutData()->pos()`.

Reply:
Ah yes - I hadn't noticed that `hook->pos()` included the offset, thanks!